torch/_inductor/lowering.py

```diff
@@ -2018,7 +2018,6 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten._thnn_fused_lstm_cell, require_dense)
 make_fallback(aten.topk)
 make_fallback(aten.upsample_bicubic2d_backward, require_contiguous)
-make_fallback(aten._scaled_mm.default)
```

Comment:
I feel we probably need to keep this since we haven't added lowering for _scaled_mm? (I'm a bit confused about this as well). Have you tried that `test/inductor/test_fp8.py` can run successfully if we remove this fallback?

Reply:
yep, I've checked and the test passes

not entirely sure either about how the fallback mechanism works though